server/stt/stt_parse_businesswire.py

```diff
@@ -0,0 +1,174 @@
+
+        # Extract SlugLine
+        slugline = xml.findtext(
+            "NewsItem/NewsComponent/NewsComponent/NewsLines/SlugLine"
+        )
+        if slugline:
+            item["slugline"] = slugline.strip()
+
+        # Extract ByLine
+        byline = xml.findtext("NewsItem/NewsComponent/NewsComponent/NewsLines/ByLine")
+        if byline:
+            item["byline"] = byline.strip()
+
+        # Extract Dateline
+        dateline = xml.findtext(
+            "NewsItem/NewsComponent/NewsComponent/NewsLines/DateLine"
+        )
+        if dateline:
+            item["dateline"] = dateline.strip()
+
+        components = xml.findall("NewsItem/NewsComponent/NewsComponent/NewsComponent")
+        for component in components:
+            role = component.find("Role")
+            if role is None:
+                continue
+
+            role_name = role.get("FormalName")
+            dest = self.COMPONENT_ROLE_MAPPING.get(role_name)
+            if not dest:
+                continue
+
+            # Extract <body> for HTML content
+            body = component.find(
+                "ContentItem/DataContent/xhtml:html/xhtml:body", namespaces=NS
+            )
+
+            if dest == "headline":
+                # Headline as plain text - store as both headline and name
+                if body is not None:
+                    headline_text = etree.tostring(
+                        body, encoding="unicode", method="text"
+                    ).strip()
+                    item[dest] = headline_text
+                    # Also store as name for compatibility
+                    item["name"] = headline_text
+
+            elif dest == "abstract":
+                # Abstract as plain string (no XHTML expected)
+                abstract = component.find("ContentItem/DataContent")
+                if abstract is not None and abstract.text:
+                    item[dest] = abstract.text.strip()
+
+            elif dest == "body_html":
+                # Join all XHTML elements in <body>
+                if body is not None:
+                    item[dest] = "\n".join(
+                        [
+                            etree.tostring(
+                                elem, encoding="unicode", method="html"
+                            ).replace(' xmlns="http://www.w3.org/1999/xhtml"', "")
+                            for elem in body
+                        ]
+                    )
+
+        # Flatten bw_keywords into keywords
+        bw_keywords = item.get("extra", {}).get("bw_keywords", {})
+        flat_keywords = [kw for group in bw_keywords.values() for kw in group]
+        if flat_keywords:
+            item["keywords"] = flat_keywords
+
+        # Extract Subject tags
+        subjects = []
+        for subj in xml.findall(".//Subject"):
+            formal = subj.get("FormalName")
+            if formal:
+                subjects.append(formal)
+        if subjects:
+            item["subject"] = subjects
```

Comment:
subjects should be stored as dicts with `name` and `qcode`, possibly also `scheme` if that should be later visible in the UI and shouldn't be mixed with other subjects